### PR TITLE
Remove doc/sample references to preview bundle

### PR DESCRIPTION
**Pull request by Charles-Gagnon**
Closes https://github.com/Azure/azure-functions-sql-extension/issues/743

Not to be merged until after GA bundle is released

### CODE REVIEW

File: docs/GeneralSetup.md [hunk 1]
```diff
@@ -153,21 +153,21 @@ These steps can be done in the Terminal/CLI or with PowerShell.
...
     ```
 
-    **JavaScript, TypeScript, Python, Powershell:** Update the `host.json` file to the preview extension bundle.
+    **JavaScript, TypeScript, Python, Powershell:** Update the `host.json` file to the v4 version of the extension bundle.
 
     ```json
     "extensionBundle": {
-        "id": "Microsoft.Azure.Functions.ExtensionBundle.Preview",
+        "id": "Microsoft.Azure.Functions.ExtensionBundle",
         "version": "[4.*, 5.0.0)"
```
Comment (MaddyDev) [inline docs/GeneralSetup.md:161]: need to update the version as well for Microsoft.Azure.Functions.ExtensionBundle to [3.*, 4.0.0)

Comment (Charles-Gagnon) [reply docs/GeneralSetup.md:161]: Is there a reason we'd want to be in the v3 of the bundle and not v4?

Comment (MaddyDev) [reply docs/GeneralSetup.md:161]: oops I thought the latest ExtensionBundle is V3 (when I created a function through func init it has the old default values) :| Never mind then 👍

**Comment by chlafreniere** [comment]
@Charles-Gagnon is it time 😄

File: docs/GeneralSetup.md [hunk 2]
```diff
@@ -153,7 +153,7 @@ These steps can be done in the Terminal/CLI or with PowerShell.
     dotnet add package Microsoft.Azure.WebJobs.Extensions.Sql --prerelease
     ```
 
-    **JavaScript, TypeScript, Python, Powershell:** Update the `host.json` file to the v4 version of the extension bundle.
+    **JavaScript, Javascript, Python, Powershell:** Update the `host.json` file to the v4 version of the extension bundle.
```
Comment (MaddyDev) [inline docs/GeneralSetup.md:156]: Javascript is noted twice.

**Review by MaddyDev — APPROVED** [review]

**Review by lucyzhang929 — APPROVED** [review]

File: docs/GeneralSetup.md (hunk 2 above)
Comment (Charles-Gagnon) [reply docs/GeneralSetup.md]: Oops, thanks. Sometimes I wonder what my brain is doing when I make changes like this...

**Pull request merged**